SPECS/python-tensorflow-estimator/python-tensorflow-estimator.spec

```diff
@@ -0,0 +1,65 @@
+%global pypi_name tensorflow-estimator
+%global _description %{expand:
+A high-level TensorFlow API that greatly simplifies machine learning programming}
+%define _enable_debug_package 0
+%global debug_package %{nil}
+
+Summary:        A high-level TensorFlow API that greatly simplifies machine learning programming
+Name:           python-%{pypi_name}
+Version:        2.11.0
```

Comment:
Are you sure this version is correct. The previous spec file has the exact same version number.

Reply:
TensorFlow, Tensorboard, Keras, and most of the dep of Tensorflow made by the TensorFlow team have the same version. For example in this case 2.11.0